coveralls/git.py

```diff
@@ -94,7 +94,8 @@ def git_info():
         }]
         if not all(head.values()):
             log.warning('Failed collecting git data. Are you running '
-                        'coveralls inside a git repository?', exc_info=ex)
+                        'coveralls inside a git repository? Is git installed?',
+                        exc_info=True)
```

Comment:
Huh, TIL you can pass in `True` instead of the exception instance. Looks like this causes an extra call to `sys.exc_info` though, which is needless when we could just use the already-captured `ex` from above. Could you revert that part of your change?

Reply:
IMO it's cleaner this way, but sure. Will push a commit in a few minutes!